Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
1. This will only be scoped to this function?
2. I might not have been clear before... on PS 5.1.14393.693 I'm seeing an error record even with `Continue`:
   <img width="408" alt="powershell_2017-01-17_08-58-25" src="https://cloud.githubusercontent.com/assets/133987/22025520/7294b3c4-dc93-11e6-849f-f82cc5e96b92.png">